src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -490,7 +504,12 @@ void generateOverallSummaryFiles(Map<Integer, Map<String, BigDecimal>> roundTall
     generateSummaryJson(roundTallies, null, outputPath);
   }
 
+  // create NIST Common Data Format CVR json
   void generateCdfJson(List<CastVoteRecord> castVoteRecords) throws IOException {
+
```

Comment:
super nit: if you don't mind, let's be consistent about not having a blank line to start methods

Reply:
if you keep up writing comments in code I am more than happy to oblige :)

Reply:
Solid whataboutism!